app-modules/contract-management/src/Filament/Resources/ContractResource.php

```diff
@@ -58,7 +57,7 @@ class ContractResource extends Resource
 
     public static function canAccess(): bool
     {
-        return ContractManagement::active() && parent::canAccess();
+        return parent::canAccess();
     }
```

Comment:
This whole override can now be removed since it is now just calling the parent.

Reply:
Done.